dex-mock/src/lib.rs

```diff
@@ -25,24 +25,25 @@ pub trait DexMock {
     fn swap_tokens_fixed_input(
         &self,
         #[payment_token] _token_in: TokenIdentifier,
-        #[payment_amount] amount_in: Self::BigUint,
+        #[payment_amount] amount_in: BigUint,
         token_out: TokenIdentifier,
-        _amount_out_min: Self::BigUint,
-        #[var_args] opt_accept_funds_func: OptionalArg<BoxedBytes>,
-    ) -> FftTokenAmountPair<Self::BigUint> {
+        _amount_out_min: BigUint,
+        #[var_args] opt_accept_funds_func: OptionalArg<ManagedBuffer>,
+    ) -> FftTokenAmountPair<Self::Api> {
```

Comment:
Return value changed to EsdtTokenPayment. But since this is a porting PR, the change can also be made on a subsequent PR

Reply:
The return value is ignored in the caller SC anyway, but I'll change it in a future PR.